plugins/spamassassin.js

```diff
@@ -20,11 +24,11 @@ exports.load_spamassassin_ini = function () {
         max_size:     500000,
         old_headers_action: "rename",
         subject_prefix: "*** SPAM ***",
-        bypass_extra_headers: false,
+        add_headers: true,
     };
```

Comment:
works for me... I first used an array similar to the one which casts numbers from string to Number but then I changed and forgot to remove the default value.